Sources/mtlswift/Utils/SourceStringBuilder.swift

```diff
@@ -28,4 +28,22 @@ public class SourceStringBuilder {
     public func add(rawString: String) {
         result += rawString
     }
+    
+    public func sanitizeResult() {
+        self.result = self.result
+                          .replacingOccurrences(of: "simd_float2,", with: "SIMD2<Float>,")
```

Comment:
why include commas and brackets into replacement?

Reply:
to ignore float3x3

Reply:
the pr looks hacky, but does the job